packages/components/src/DateTimePickers/Time/Input/Input.component.js

```diff
@@ -8,9 +9,10 @@ import theme from './Input.scss';
 
 export default function Input(props) {
 	const { time, inputManagement } = useContext(TimeContext);
+	const { minWidth } = props;
```

Comment:
please destructure the minwidth so it is not spread to the debounce input and only consume by the sizer

Reply:
done, concerning the question about the +5px when setting minwidth, this is because the inputSizer adds an extra space at the end.